tests/landsat8_vsir.flt

```diff
@@ -0,0 +1,1061 @@
+N
```

Comment:
Do we really need another copy of the spectral response? Can't we use the files under _wagl/spectral_response_ ?